src/clj/rems/ga4gh.clj

```diff
@@ -34,10 +37,12 @@
    :sub s/Str
    :exp s/Int
    :iat s/Int
-   :scope (s/eq "openid") ;; could also list multiple space-separated scopes, as long as one is "openid"
+   (s/optional-key :scope) s/Str ;; Embedded Document Tokens "MUST NOT contain 'openid'"
```

Comment:
This would be the right place for `;`

Reply:
I don't like how emacs indents `;`....

Reply:
hmm no wait that's changed, it no longer indents the `;` to a fixed column

Reply:
it's only if you put it on a newline?